### ISSUE TITLE
Submission Notification

### ISSUE THREAD

**Issue by DLibatique** [DLibatique/CLAS199-old#4]
At the point of submission, I'd like to be notified about the submission and who submitted it, and per our fb messenger chat, it would be best not to get an e-mail notification for every single submission. Maybe we can set up a Google Sheet or something, with a list of student names -- every time the student submits a document (as evidenced by the name that they'd type alongside the submission -- see issue #2), maybe we can add a tick next to their name?

**Comment by shadowlion** [comment]
@DLibatique definitely a lot more work then initially thought lol - but it's a feature nevertheless

**Comment by DLibatique** [comment]
@shadowlion nah, then, we can think of something else -- no need to make things more complicated than necessary. I might even just do it manually, i.e. check every Friday or something
